db/sstables-format-selector.cc

```diff
@@ -24,60 +24,75 @@ static const sstring SSTABLE_FORMAT_PARAM_NAME = "sstable_format";
 void feature_enabled_listener::on_enabled() {
     if (!_started) {
         _started = true;
-        _selector.maybe_select_format(_format).get();
```

Comment:
We should probably have a test to check that sstables created during commitlog replay on startup are created in the right (newest) format.

Can be left to a follow-up.

Reply:
ok, I'll add it